src/Microsoft.ML.Data/Model/Onnx/OnnxContext.cs

```diff
@@ -2,245 +2,103 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
 using System.Collections.Generic;
-using System.Linq;
-using Microsoft.ML.Runtime.UniversalModelFormat.Onnx;
 using Microsoft.ML.Runtime.Data;
 
 namespace Microsoft.ML.Runtime.Model.Onnx
 {
     /// <summary>
-    /// A context for defining a ONNX output.
+    /// A context for defining a ONNX output. The context internally contains the model-in-progress being built. This
+    /// same context object is iteratively given to exportable components via the <see cref="ICanSaveOnnx"/> interface
+    /// and subinterfaces, that attempt to express their operations as ONNX nodes, if they can. At the point that it is
+    /// given to a component, all other components up to that component have already attempted to express themselves in
+    /// this context, with their outputs possibly available in the ONNX graph.
+    /// 
+    /// 
```

Comment:
you probably don't need this lines.

Reply:
Probably not. But you never know.

---
In reply to: [199963584](https://github.com/dotnet/machinelearning/pull/462#discussion_r199963584) [](ancestors = 199963584)